packages/site-admin/src/components/page/style.scss

```diff
@@ -0,0 +1,64 @@
+@import "@wordpress/base-styles/variables";
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/z-index";
+
+.a8c-site-admin-page {
+	color: $gray-800;
+	background: $white;
+	height: calc(100% - #{$header-height});
+	container: a8c-site-admin-page / inline-size;
+
+	@media not (prefers-reduced-motion) {
+		transition: width ease-out 0.2s;
```

Comment:
When do we animate the page's `width` ? Animating the `width` of an element usually leads to janky animations, especially for a container like page where all of it children will require to be laid out continuosly during the animation

Reply:
removed